core/cowdata.h

```diff
@@ -53,33 +53,30 @@ class CowData {
 private:
 	mutable T *_ptr;
 
-	// internal helpers
+	/* Internal Helpers */
```

Comment:
Just being a bit nit-picky here, but why the change to the comment style? It's a single-line comment, shouldn't it be `// comment`? Forgive me if you're seeing something I'm not.

Reply:
You're right that there is no real difference here, but I was just testing a comment style system that I mention in issue #21696. At this stage I'm just looking for feedback on this issue, and believe code reviewers are in a good position to objectively judge a system like this :)
Ultimately this pr isn't about the comment style and I can revert it back to a single line comment, if that is desired.